packages/firebase/index.d.ts

```diff
@@ -1119,7 +1119,7 @@ declare namespace firebase.app {
      *   });
      * ```
      *
-     * @return {!firebase.Promise<void>} An empty promise fulfilled when the app has
+     * @return {!Promise<void>} An empty promise fulfilled when the app has
```

Comment:
Please remove the type, so it will be just `*  @return An empty promise fulfilled when the app has....`

Reply:
Done. vim macros.